When building for avr32, the build fails as follows.

  cc1: error: unrecognized command line option "-Wno-packed-bitfield-compat"

An example of an autobuild failure arising from this is the following.

  http://autobuild.buildroot.net/results/92e/92e472004812a3616f62d766a9ea07a997a66e89/

Clearly, not all toolchains provide a gcc that understands
the -Wno-packed-bitfield-compat flag; remove usage of this flag.

Signed-off-by: Simon Dawson <spdawson@gmail.com>

diff -Nurp a/util/scan/Makefile b/util/scan/Makefile
--- a/util/scan/Makefile	2013-11-24 17:04:10.000000000 +0000
+++ b/util/scan/Makefile	2013-12-13 09:37:11.967975173 +0000
@@ -14,7 +14,7 @@ inst_bin = $(binaries)
 
 removing = atsc_psip_section.c atsc_psip_section.h
 
-CPPFLAGS += -Wno-packed-bitfield-compat -D__KERNEL_STRICT_NAMES
+CPPFLAGS += -D__KERNEL_STRICT_NAMES
 
 .PHONY: all
 
